tests/integration/synapseclient/core/test_download.py

```diff
@@ -343,6 +348,175 @@ async def test_download_expired_url(
         # THEN the file is downloaded anyway AND matches the original
         assert filecmp.cmp(original_file, path), "File comparison failed"
 
+    async def test_download_expired_presigned_url(
+        self,
+        syn: Synapse,
+        mocker: MockerFixture,
+        project_model: Project,
+    ) -> None:
+        """Tests that a file download is retried with a presigned URL but it is expired."""
+        # GIVEN a file stored in synapse
+        original_file = utils.make_bogus_data_file(40000)
```

Comment:
Could this be made smaller (Does the default of 100 work?). The reason is that I want to ensure the tests need to transfer as little data as possible so they run quicker